components/courses/spec/services/courses/test_task/create.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe Courses::TestTask::Create do
+  let!(:season)          { create(:season, title: 'Test Season') }
+  let!(:user)            { create(:user) }
+  let!(:student)         { ::Courses::Student.create(season_id: 1, user_id: 1, status: :enrolled,
```

Comment:
@denysmedynskyi I will fix it myself. We need asap finish dev today and open huge PR into development